controllers/root.go

```diff
@@ -14,6 +18,16 @@ type Context struct {
 	Settings *helpers.Settings
 }
 
+var basePath = os.Getenv("BASE_PATH")
+var templates = template.Must(template.ParseFiles(filepath.Join(basePath, "static/index.html")))
```

Comment:
also, if we do the above and put the extra env var, when we run `InitApp`, we can add a member variable to `Context` named `templates` and when we initialize Context, we can assign the template there instead of here as a global variable to controller package